docs/topics/ref_known-issues-7-1-0.adoc

```diff
@@ -10,4 +10,4 @@ This section provides highlighted known issues in {ProductFullName} version 7.1.
 
 .Enabling Preemption does not work
 
-In {ProductFullName} 7.1.0, the *Preemption* option cannot be enabled for tasks in the Task Manager page. Currently, there is no workaround available. 
+In {ProductFullName} 7.1.0, the *Preemption* option cannot be enabled for tasks in the Task Manager page. Currently, there is no workaround available. (link:https://issues.redhat.com/browse/MTA-3195[MTA-3195])
```

Comment:
@mpershina enabling preemption does not work also in the task manager drawer.

Reply:
Oki, added a suggestion below